hooks/settings.py

```diff
@@ -29,169 +28,11 @@ def get_export_preset(self):
         
         :returns: Path on disk to Flame export preset 
         """
-        return self._generate_profile_xml()
-
-    ###############################################################################################
-    # helper methods and internals
-    
-    def _write_content_to_file(self, content, file_name):
-        """
-        Helper method. Writes content to file and returns the path.
-        The content will be written to the app specific cache location 
-        on disk, organized by app instance name. The rationale is that 
-        each app instance holds its own configuration, and the configuration
-        generates one set of unique xml files.
-        
-        :param content: Data to write to the file
-        :param file_name: The name of the file to create
-        :returns: path to the created file
-        """
-        # determine location
-        folder = os.path.join(self.parent.cache_location, self.parent.instance_name)
-        file_path = os.path.join(folder, file_name)
-
-        # create folders
-        if not os.path.exists(folder):
-            old_umask = os.umask(0)
-            os.makedirs(folder, 0777)
-            os.umask(old_umask)
-        
-        # write data
-        fh = open(file_path, "wt")
-        fh.write(content)
-        fh.close()
-        
-        self.parent.log_debug("Wrote temporary file '%s'" % file_path)
-        return file_path
-        
-    def _generate_quicktime_settings(self):
-        """
-        Generate quicktime codec presets and return the path to 
-        the generated file.
-        
-        :returns: path to quicktime codec presets (.cdxprof) xml file
-        """ 
+        # base it on one of the default presets that ship with Flame.
+        return os.path.join(
+            self.parent.engine.install_root,
+            "presets",
+            self.parent.engine.flame_version,
+            "export/presets/flame/movie_file/QuickTime (H.264 720p 8Mbits).xml"
```

Comment:
would it make sense to access this path (or the presets directory) via the engine?